raiden/tests/integration/cli/test_cli.py

```diff
@@ -6,8 +6,11 @@
 from click.testing import CliRunner
 from eth_utils import to_checksum_address
 
+from raiden.constants import Environment
 from raiden.ui.cli import run
 
+EXPECTED_DEFAULT_ENVIRONMENT_VALUE = Environment.PRODUCTION.value.lower()
```

Comment:
The `value` is already lowercase.. if `EnvironmentChoiceType` is waiting on valid string values, you don't need and shouldn't change it here, as the `Enum` defines the domain of expected/valid values.